src/Indicator.vala

```diff
@@ -105,33 +149,31 @@ public class DateTime.Indicator : Wingpanel.Indicator {
         var events = Widgets.CalendarModel.get_default ().get_events (calendar.selected_date);
         if (events.size == 0) {
             update_events_idle_source = 0;
+            no_events_label.visible = true;
```

Comment:
Instead of this you can use ListBox's native placeholder: https://valadoc.org/gtk+-3.0/Gtk.ListBox.set_placeholder.html

Reply:
If I do that, the events pane will never show the `no_events_label` label, even if I rewrite this to be a placeholder. Something awry in the backend, I suppose.